src/arrays.js

```diff
@@ -1,61 +1,65 @@
 const getNthElement = (index, array) => {
-  // your code here
+index %= array.length;
+return array[index];
 };
 
 const arrayToCSVString = array => {
-  // your code here
+return array.join();
 };
 
 const csvStringToArray = string => {
-  // your code here
+  return string.split (',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+array.push(element);
```

Comment:
This should be indented 2 spaces. The same goes for the rest of the functions in this file, so I would advise going through it and fixing this for all of them rather than me writing each one out again here:
```
  array.push(element);
```